diagrams/request_uri_mode_post.md

```diff
@@ -0,0 +1,45 @@
+```plantuml
+@startuml
+
+autonumber
+
+box "Wallet"
+participant "Metadata" as wm
+participant "Authorization Endpoint" as w
+end box
+
+participant "User Agent" as u
+
+box "Verifier"
+participant "Frontend" as r
+participant "Request Endpoint" as rp
+participant "Response Endpoint" as rb
+end box
+
+u --> r : use
+activate r
+
+r --> u: authorization request\n(client_id, request_uri, request_uri_mode=POST)
```

Comment:
```suggestion
r --> u: authorization request\n(client_id, request_uri, request_uri_mode=POST, [client_id_scheme])
```

Reply:
client_id_scheme allows the Wallet to establish the trust with the verifier before anything data might be sent to it